linkerd/app/src/oc_collector.rs

```diff
@@ -68,7 +69,7 @@ impl Config {
                     }))
                     .push(control::add_origin::Layer::new())
                     .into_new_service()
-                    .new_service(addr.clone());
+                    .with_fixed_target(addr.clone());
```

Comment:
This seemed like a better approach to me than making `linkerd2-opencensus` depend on `linkerd2-app-core` just to get the `ControlAddr` type, wdyt?